docs/CustomServices.md

```diff
@@ -28,22 +26,67 @@ export default class CustomService {
 }
 ```
 
-An Error thrown during a service hook will be logged while the runner continues. If a hook in your service is critical to the setup or teardown of the test runner, the `SevereServiceError` exposed from the `webdriverio` package can be used to stop the runner. 
+Whereas a worker service should look like this:
+
+```js
+export default class CustomWorkerService {
+    /**
+     * `serviceOptions` contains all options specific to the service
+     * e.g. if defined as follows:
+     *
+     * ```
+     * services: [['custom', { foo: 'bar' }]]
+     * ```
+     *
+     * the `serviceOptions` parameter will be: `{ foo: 'bar' }`
+     */
+    constructor (serviceOptions, capabilities, config, browser) {
+        this.browser = browser
+    }
+
+    before(config, capabilities) {
+        // TODO: something before all tests are run, e.g.:
+        this.browser.setWindowSize(1024, 768)
+    }
+
+    after(exitCode, config, capabilities) {
+        // TODO: something after all tests are run
+    }
+
+    // other hooks or custom service methods ...
+}
+```
+
+It is recommended to store the browser object through the passed in paramater in the constructor. Lastly expose both types of workers as following:
+
+```js
+import CustomLauncherService from './launcher'
+import CustomWorkerService from './service'
+
+export default CustomWorkerService
+export const launcher = CustomLauncherService
+```
+
+## Service Error Handling
+
+An Error thrown during a service hook will be logged while the runner continues. If a hook in your launcher service is critical to the setup or teardown of the test runner, the `SevereServiceError` exposed from the `webdriverio` package can be used to stop the runner.
```

Comment:
Should we add an example on how to stop the runner from within a server?

Reply:
What do you mean with `from within a server`?

Reply:
I meant from within a service, it was late haha